### PR TITLE
Update realtime_servo_tutorial.rst

### PR DESCRIPTION
**Pull request by devvaibhav455**
### Description

Although, the code suggests cloning the old version of UR's repo but it does not clearly mention it in words. I did not clone the old version because I already had the latest version and the tutorial didn't work for me for now obvious reasons. So, I am adding a note in bold so that no one else does the same mistake as mine and don't waste 2 days like me. I also added a debug tip to see when one should expect the tutorial to work.

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: doc/realtime_servo/realtime_servo_tutorial.rst
Comment (AndyZe) [inline doc/realtime_servo/realtime_servo_tutorial.rst:22]: I'd recommend deleting this link. I don't think it adds anything, really, and certainly needs a better description than "Src". Looks good to me besides that. Thanks!

```suggestion
**NOTE: Please stick to older version of Universal Robot Github (1.2.7 ) as newer versions (1.3.x) are incompatible with the servo tutorial.**
```

**Comment by devvaibhav455** [comment]
Hello Dr. @AndyZe  : I made the suggested changes. Were you able to check them? Happy July 4 btw.

**Review by AndyZe — APPROVED** [review]

File: doc/realtime_servo/realtime_servo_tutorial.rst
Comment (AndyZe) [inline doc/realtime_servo/realtime_servo_tutorial.rst:136]:
```suggestion
```

File: doc/realtime_servo/realtime_servo_tutorial.rst
Comment (AndyZe) [inline doc/realtime_servo/realtime_servo_tutorial.rst:132]:
```suggestion
 
```

File: doc/realtime_servo/realtime_servo_tutorial.rst
Comment (AndyZe) [inline doc/realtime_servo/realtime_servo_tutorial.rst:132]:
```suggestion

```

File: doc/realtime_servo/realtime_servo_tutorial.rst
Comment (rhaschke) [inline doc/realtime_servo/realtime_servo_tutorial.rst:134]: This should be validated and correspondingly rephrased before merging.

**Review by rhaschke — COMMENTED** [review]
Please verify that `joint_group_velocity_controller` works as well and rephrase the debug comment.

File: doc/realtime_servo/realtime_servo_tutorial.rst
Comment (AndyZe) [inline doc/realtime_servo/realtime_servo_tutorial.rst:134]:
```suggestion
Do run either ``rosservice call /controller_manager/list_controllers`` or ``rosrun controller_manager controller_manager list``. For moveit_servo to be working, ``joint_group_position_controller`` or ``joint_group_velocity_controller`` should be ``running``.
```

File: doc/realtime_servo/realtime_servo_tutorial.rst
Comment (AndyZe) [inline doc/realtime_servo/realtime_servo_tutorial.rst:133]:
```suggestion
  catkin run_tests --this

Debug Tips
---------------
```

**Review by rhaschke — APPROVED** [review]
Approving if .rst formatting issue reported by CI is fixed.

**Pull request merged**
